### PR TITLE
Merge test results and send to coveralls

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

File: rakelib/test_all.rake [hunk 1]
```diff
@@ -145,6 +159,10 @@ task :test_all do
   end
 
   if exceptions.empty?
+    r.delete_if { |path, coverage| path.to_s.match(/spec/) }
```
Comment (KrauseFx) [inline rakelib/test_all.rake]: Nice

Comment (neonichu) [inline rakelib/test_all.rake]: 👌

**Comment by KrauseFx** [comment]
Can't merge since code coverage is failing. I didn't enable failing builds on coveralls, I'm confused

**Comment by KrauseFx** [comment]
Let's try this

**Comment by neonichu** [comment]
MAKE FASTLANE GREAT AGAIN 🤗

**Pull request merged**